app/lib/features/home/widgets/my_spaces_section.dart

```diff
@@ -101,22 +98,22 @@ class _NoSpacesWidget extends ConsumerWidget {
           text: TextSpan(
             children: <TextSpan>[
               TextSpan(
-                text: L10n.of(context).createOrJoinSpace('create'),
+                text: L10n.of(context).create,
                 style: Theme.of(context).textTheme.bodyMedium!.copyWith(
                       decoration: TextDecoration.underline,
                       fontWeight: FontWeight.bold,
                     ),
               ),
               TextSpan(text: '\t${L10n.of(context).or}\t'),
               TextSpan(
-                text: L10n.of(context).createOrJoinSpace('join'),
+                text: L10n.of(context).join,
                 style: Theme.of(context).textTheme.bodyMedium!.copyWith(
                       decoration: TextDecoration.underline,
                       fontWeight: FontWeight.bold,
                     ),
               ),
               TextSpan(
-                text: '\t${L10n.of(context).createOrJoinSpace('restText')}',
+                text: '\t${L10n.of(context).spaceShortDescription}',
```

Comment:
what formatting weirdness is this? why is there a forced tab? Is that how we do design now? Let's remove this, while we are at it.
```suggestion
                text: L10n.of(context).spaceShortDescription,
```